css/style.css

```
@@ -0,0 +1,106 @@
+/***************************/
+/* HEADER */
+/**************************/
+.header {
+  display: flex;
+  padding-left: 18rem;
+  padding-right: 18rem;
+  position: absolute;
+  gap: 2.2rem;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+}
+
+.promo {
+  background-image: url("/img/promo-background.png");
+  box-shadow: inset 0px 140px 40px rgb(0 0 0 / 30%);
+  min-height: 75rem;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  margin-left: auto;
+  color: #fff;
+  font-size: 3.6rem;
+  height: 12rem;
+  width: 100%;
+}
+
+.nav {
+  display: flex;
+  align-self: stretch;
+}
+
+.header-nav-list {
+  display: flex;
+  list-style: none;
+  gap: 0.8rem;
+}
```

This block have to take all the height available
```suggestion
  gap: 0.8rem;
  height: 100%;
}
```